microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -656,9 +656,9 @@ def _execute_xfstests(
         # test_group
         # a test group for XFS will fail for a config for ext or btrfs
         test_group: str = ""
-        if not test_type:
-            test_group = f"{file_system.name}/{test_type}"
-        if test_type == "generic":
+        if not test_type or test_type == file_system.name:
+            test_group = f"{file_system.name}/quick"
+        elif test_type == "generic":
```

Comment:
The default value of test_type is `generic/quick`, and the test_type is given in your own code. So, it shouldn't be set to `generic`.